aim/agent/aid/universes/aci/converter.py

```diff
@@ -1149,6 +1153,22 @@ def bgp_as_id_converter(object_dict, otype, helper, to_aim=True):
                        'converter': fvRsCons_Ext_converter,
                        'convert_pre_existing': True,
                        'convert_monitored': False}],
+    'infraRsSpanVSrcGrp': [{'resource': resource.InfraAccBundleGroup,
```

Comment:
Are these needed? Weren't they already added to the initial resource_map above?

Reply:
You are right, that was duplicated. Removed it in the lastest commit.